migtests/scripts/resumption.py

```diff
@@ -191,16 +194,22 @@ def run_command(command, allow_interruption=False, interrupt_after=None):
             print("\nCommand Output:\n")
             for line in stdout.splitlines():
                 print(line)
-
         if stderr:
```

Comment:
Are we printing stderr  for last execution only?

Reply:
No it will print for intermediate runs as well if there is an stderr.